### PR TITLE
Update roadmap.html

### PR DESCRIPTION
**Pull request by saimaghafoor**
Update to roadmap

### CODE REVIEW

File: app/templates/views/guidance/features/roadmap.html [hunk 1]
```diff
@@ -28,14 +28,14 @@ <h1 class="heading-large">Roadmap</h1>
...
-    <li>Let services attach pages to letter templates</li>
+    
+    <li>Create a <a class="govuk-link govuk-link--no-visited-state" href="https://docs.google.com/forms/d/1Fp9krEODWMnMW6mG_UfWa7zzXTKTHLuTUCyBR2QoxZ0/edit">newsletter you can subscribe to</a></li>
+    <li>Let services add QR codes to letter templates</li>   
     <li>Let services create letter templates that comply with the <a class="govuk-link govuk-link--no-visited-state" href="https://law.gov.wales/culture/welsh-language/welsh-language-act-1993">Welsh Language Act 1993</a></li>
-    <li>Let services add QR codes to letter templates</li>
     <li>Integrate with our letter provider’s API to improve the letter sending process</li>
     <li>Add more formatting options to email and letter templates</li>
     <li>Continue to improve our front-end code to meet our <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.accessibility_statement')}}">targets for accessibility</a></li>
+    <li>Give users the option to set a custom data retention period that's more than 7 days</li>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/features/roadmap.html:38]:
```suggestion
    <li>Give users the option to set a custom data retention period that’s more than 7 days</li>
```

File: app/templates/views/guidance/features/roadmap.html [hunk 2]
```diff
@@ -58,6 +58,10 @@ <h2 class="heading-medium" id="things-we-have-done">Things we’ve done</h2>
 
   <ul class="govuk-list govuk-list--bullet">
 
+    <li>Let services attach pages to letter templates</li>
+    <li>Give organisations permission to approve new services</li>
+    <li>Users can consent to take part in research</li>
+    <li>Introduce subheadings for emails</li>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/features/roadmap.html:64]:
```suggestion
    <li>Introduce subheadings for emails</li>
```

Link to https://www.notifications.service.gov.uk/using-notify/formatting

or https://www.notifications.service.gov.uk/using-notify/formatting#headings-and-subheadings if we think it’s ok to deep link to an anchor

**Comment by saimaghafoor** [comment]
Good to go @tombye

File: app/templates/views/guidance/features/roadmap.html [hunk 3]
```diff
@@ -15,8 +15,8 @@ <h1 class="heading-large">Roadmap</h1>
 
   {{ content_metadata(
     data={
-      "Last updated": "2 May 2023",
-      "Next review due": "27 July 2023"
+      "Last updated": "1 August 2023",
```
Comment (tombye) [inline app/templates/views/guidance/features/roadmap.html:18]:
```suggestion
      "Last updated": "2 August 2023",
```

**Review by tombye — CHANGES_REQUESTED** [review]
Just needs the review date bumped, otherwise looks good to go 👍🏻

**Review by tombye — APPROVED** [review]
Good to go 🚀 👍🏻

**Pull request merged**